dotnet/Directory.Packages.props

```diff
@@ -5,7 +5,7 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="Azure.AI.OpenAI" Version="1.0.0-beta.12" />
+    <PackageVersion Include="Azure.AI.OpenAI" Version="1.0.0-beta.13" />
```

Comment:
Hopefully we can remove this now?
https://github.com/microsoft/semantic-kernel/blob/eed51932f2fb5879de47e76b752f9051cb868331/dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/OpenAIFunctionToolCall.cs#L172-L179

Reply:
Done